rkt/config/auth.go

```diff
@@ -59,7 +75,7 @@ type basicAuthHeaderer struct {
 	auth basicV1
 }
 
-func (h *basicAuthHeaderer) Header() http.Header {
+func (h *basicAuthHeaderer) GetHeader() http.Header {
```

Comment:
@bwalex I'm guessing you changed the name to convey that there are some underlying calculations to get the value, but still I'd prefer we didn't re-introduce getters